pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -249,3 +287,57 @@ func (b *BridgePodInterface) createDefaultBridge() error {
 
 	return nil
 }
+
+type SlirpPodInterface struct {
+	iface           *v1.Interface
+	network         *v1.Network
```

Comment:
I can't see where is this being used. Why do we need network here?

Reply:
If both have structs have the same attributes perhaps it can simply be 
```
type SlirpPodInterface struct {
	BridgePodInterface
}
```

Reply:
I will remove the network variable.